scribejava-httpclient-armeria/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,29 @@
+# SLF4J's SimpleLogger configuration file
+# Simple implementation of Logger that sends all enabled log messages, for all defined loggers, to System.err.
```

Comment:
Does it really needed?

Reply:
No, it's not necessary. I'll remove it with an updated PR.